app/assets/javascripts/Components/Result/marks_panel.jsx

```diff
@@ -343,16 +345,16 @@ class RubricCriterionInput extends React.Component {
 
     return (
       <tr
-        data-level-index={i} onClick={() => this.handleChange(i)}
-        key={`${this.props.id}-${i}`}
+        onClick={() => this.handleChange(levelIndex)}
```

Comment:
@ashih2018 let's change `this.handleChange` so that it *also* accepts a level object. This will allow you to simplify the code and remove the calculation of `levelIndex` as well.